dev/bots/test.dart

```diff
@@ -32,7 +32,7 @@ final String pubCache = path.join(flutterRoot, '.pub-cache');
 final String toolRoot = path.join(flutterRoot, 'packages', 'flutter_tools');
 
 /// The arguments to pass to `flutter test` (typically the local engine
-/// configuration) -- prefilled with the arguments passed to test.dart.
+/// configuration) -- pre-filled with the arguments passed to test.dart.
```

Comment:
"prefill" is more common than "pre-fill"

https://books.google.com/ngrams/graph?content=prefill%2C+pre-fill&year_start=1800&year_end=2000&corpus=15&smoothing=3&share=&direct_url=t1%3B%2Cprefill%3B%2Cc0%3B.t1%3B%2Cpre%20-%20fill%3B%2Cc0#t1%3B%2Cprefill%3B%2Cc0%3B.t1%3B%2Cpre%20-%20fill%3B%2Cc0